modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc

```diff
@@ -36,48 +37,170 @@ Showcase apps demonstrate many features of {product-name}. To see these features
 
-.. Provision a PostgreSQL database if there isn't one available.
-... Login to the OpenShift web console, create a new project or use an existing one
-... Click on the `Add to Project` dropdown on the top right, and choose `Browse Catalog`
-... Select `Databases` -> `Postgres` -> `PostgreSQL`, and complete the wizard to start the deployment of the database. Take notes of the values for `PostgreSQL Connection Username`, `PostgreSQL Connection Password` and `PostgreSQL Database Name`. You will need them later.
-.. Provision a new {sync-service} app from the catalog as described in xref:data-sync.adoc#sync-server-openshift[Running Data Sync in OpenShift] with the following parameters:
+// end::excludeDownstream[]
+
+== Demonstrate the {sync-service}
+
+. Ensure a PostgreSQL database is available so that you can connect to it at a later stage.
+.. Login to OpenShift and, create a new project or use an existing project.
+.. Click `Browse Catalog` and select `Databases` -> `Postgres` -> `PostgreSQL`, and complete the wizard to start the deployment of the database.
++
+NOTE: Record the value for `PostgreSQL Connection Username`, `PostgreSQL Connection Password`, and `PostgreSQL Database Name` as they are required when provisioning {sync-service}.
+.. Click `Browse Catalog` and select `Mobile` -> `Data Sync`.
+. Provision a new {sync-service} app from the catalog as described in xref:data-sync.adoc#sync-server-openshift[Running Data Sync in OpenShift] with the following parameters:
 +
 [options="header"]
++
 |====
 |Field|Value
-|Sync Application Name|syncshowcase
-|Sync Application Docker image|http://docker.io/aerogear/voyager-server-example-task
-|Sync Application health endpoint| health
-|Sync Application metrics endpoint| metrics
-|Sync App Port|4000
-|Sync App GraphQL Endpoint| graphql
-|Database Server Hostname|*Hostname of the PostgreSQL server*
-|Database Server Port|*Port number of the PostgreSQL server, usually should be 5432*
-|Database Server Username|*Username of the PostgreSQL database connection*
-|Database Server Password|*Password of the PostgreSQL database connection*
-|Database name|*PostgreSQL database name*
+|Sync Application Name| syncshowcase
+|Sync Application Docker image| docker.io/aerogear/voyager-server-example-task
+|Sync Application Docker image version| latest
+|Sync App Port| 4000
+|Sync Application metrics endpoint| /metrics
+|Sync App GraphQL Endpoint| /graphql
+|Database Server Hostname| Hostname of the PostgreSQL server, usually *postgresql*
+|Database Server Port| Port number of the PostgreSQL server, usually *5432*
+|Database Server Username| *Username of the PostgreSQL database connection*
+|Database Server Password| *Password of the PostgreSQL database connection*
+|Database name| *PostgreSQL database name*
 |====
-.. xref:data-sync.adoc#sync-server-binding-keycloak[Protect the {sync-service} app using {keycloak-service}]
++
+. Provision the Mobile Developer Console
```

Comment:
why don't we just have a prereq for https://docs.aerogear.org/aerogear/latest/mobile-developer-console.html#provision-mdc ?

Reply:
IIRC (it's been a while), I think we discussed (generally speaking) separating the showcase app from the services, and this is a consequence of this work. Another example that is part of this PR can be seen here: ["include::../generic-local-mobile-dev-envir-reqs.adoc[] " was replaced with the content within this file.](https://github.com/aerogear/mobile-docs/pull/682/commits/58c32ba8036d4e9192786a449b0e4137f636e176#diff-6548f1cccbdf3e920c68badb77e6ae10L81)